src/sklearn_evaluation/plot/grid_search.py

```diff
@@ -22,6 +23,32 @@
 )
 
 
+def _validate_change_input(change, valid):
+    if change is None:
+        raise ValueError(
+            (
+                "change can't be None, you need to select at least"
+                " one value to make the plot."
+            )
+        )
+
+    to_validate = change
+
+    if isinstance(change, str):
+        to_validate = [change]
+
+    for input in to_validate:
+        keys(
+            valid=valid,
+            passed=input,
+            name="change",
+        )
+
+
+def _validate_kind_input(kind, valid):
+    keys(valid, passed=kind, name="kind")
```

Comment:
let's switch the import to `from ploomber_core import validate` so here we have `validate.keys`, this way it'll be clearer than just calling `keys`